### PR TITLE
Containerize Chartbrew

### PR DESCRIPTION
**Pull request by Mobilpadde**
Made it a tad easier to test and deploy `chartbrew` by using docker.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+version: '3.3'
+
+services:
+  db:
+    image: mysql:5.7
+    restart: always
+    environment:
+      MYSQL_DATABASE: chartbrewdev
```
Comment (razvanilin) [inline docker-compose.yml]: All these vars should use the env variables defined in the `.env` file in the root folder like so:

```
MYSQL_DATABASE: ${CB_DB_NAME_DEV}
MYSQL_USER: ${CB_DB_USERNAME_DEV}
...
```

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+npm run setup
+npx npm-run-all -p front back
```
Comment (razvanilin) [inline entrypoint.sh]: `front` and `back` should be renamed as `client` and `server`

File: package.json [hunk 3]
```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "setup": "npm install && cd client && cp -n src/config/settings.template.js src/config/settings.js && npm install && cd ../server/ && npm install && cp -n models/.env_template models/.env && npm run db:migrate",
+    "front": "cd client && npm start",
```
Comment (razvanilin) [inline package.json]: `front` should be `client`

File: package.json [hunk 4]
```diff
@@ -6,6 +6,8 @@
...
+    "back": "cd server && npm run start-dev",
```
Comment (razvanilin) [inline package.json]: `back` should be `server` (for consistency)

**Comment by Mobilpadde** [comment]
@razvanilin should be fixed now :dancer:

**Pull request merged**